### DISCUSSION TITLE
Restricted audience + vouchers + waiting list

### DISCUSSION THREAD

**Discussion by korteke** [pretix/pretix#3727] [Q&A] [answered]
We organize events on a monthly basis and have two types of tickets. For members, and for non-members. At the moment we send one voucher code to all our members and the fastest ones then get a ticket booked. Without the voucher the waiting list works very well, but for the ticket types that require a voucher this is not possible?
The situation is that we have limited seats and significantly more members, and we would want to have waiting list enabled to our members.

So it there a option that allows members to add them self to waiting list, if they have voucher and all the tickets has already been sold?

**Comment by raphaelm** [comment, marked as answer]
No, the waiting list does not support such complex use cases currently

**Reply by korteke** [reply]
Thank you for the reply. I need to figure out something else.

**Reply by pc-coholic** [reply]
Here is an idea...

If you can manage to get all of your members to use a pretix account (or SSO), give all members a membership within pretix and restrict that ticket type to be only purchaseable with said membership, you a) do not need to send out a member-voucher and b) could still have the waitlist enabled on the members-only product...

**Reply by korteke** [reply]
Yeah, I was thinking that as an option, and probably need to go to that direction. But before that I wanted to ask if there is some other possibilities :)
But yes, that's probably the easiest way, not the nicest from UX perspective, but manageable.
